### ISSUE TITLE
Exported file not passing zugferd validations and PDF/A3 mark missing.

### ISSUE THREAD

**Issue by vucenovic** [ZUGFeRD/mustangproject#199]
Hello, we are using the ```ZUGFeRDExporterFromA3``` with an PDF-A3 file as input, fill the xml/transaction and export with zugferdversion 2 and pdfaconformance set to BASIC, however, the exported file is not marked as PDF-A3 in our PDF viewer (as our input is) and the produced pdf with attached xml does not even pass zugferd 1.0 validations. Here is our preparation code:

```ZUGFeRDExporterFromA3 exporter = new ZUGFeRDExporterFromA3();
		exporter.load(pdf);
		exporter.setXML(invoiceXML);
		ByteArrayOutputStream out = new ByteArrayOutputStream();
		exporter.setZUGFeRDVersion(2);
		exporter.setConformanceLevel(PDFAConformanceLevel.BASIC);
		exporter.export(out);
		return out.toByteArray();
```

The xml is prepared beforehand by your library. 

Do you know what could cause this issue?

**Comment by vucenovic** [comment]
The validation states that the pdf is not pdf-a3 and that the profile "xrechnung" is not valid, and therefore the validation failed.

**Comment by vucenovic** [comment]
The issue was resolved by updating to version 2.0.1, there was an error in keeping the pdf metadata of a valid pdf/a3 input.
